### ISSUE TITLE
Hunter Douglas PowerView Shades

### ISSUE THREAD

**Issue by beowulfe** [openhab/openhab2-addons#659]
Hunter Douglas makes a line of motor controlled shades, by the name of PowerView. I have a binding ready to submit for controlling these, but I have a few questions first:

1) Rollershutter items would seem like a good fit, however PowerView doesn't have up/down controls. Instead, you specify a percentage to move the shade to, or a percentage to vane open/closed the slats on a fully closed shade. Is it acceptable to instead expose two Dimmer items: one for the position, and another for the vane?

2) I first worked on this before the OH2 beta, when discovered Things would auto-create the Items for their Channels. It seems like this isn't possible now in the beta? Is the intention to leave Item creation as a manual step in the short/medium term? I just want to be clear in the documentation I write for the binding (i.e. if I should bother including manual Item creation as a step).

**Comment by kaikreuzer** [comment]
1) I would go for a Rollershutter+Dimmer item, this is also how it is e.g. done in KNX. Simply translate UP to 0% and DOWN to 100% in your binding. The more tricky command is STOP/MOVE - is it really not possible to simply stop the movement of your shades? If not, simply ignore these commands then.

2) Yes, the general assumption is that it is the user who defines the items and creates the links - the very same way as in openHAB 1. So this is what your documentation should also assume. For the "auto-creation" of items, I have entered https://github.com/eclipse/smarthome/issues/1039 - feel free to join the discussion!

**Comment by beowulfe** [comment]
1) Sounds good. Regarding STOP/MOVE, I wasn't able to discover a way to do it. Conceivably I could implement it myself by progressively updating the position, but there's no feedback mechanism to know when the position has been reached. I'll just ignore the commands.

**Comment by andrewfg** [comment]
> Regarding STOP/MOVE, I wasn't able to discover a way to do it. Conceivably I could implement it myself by progressively updating the position, but there's no feedback mechanism to know when the position has been reached. I'll just ignore the commands.

@andylintner (just so you know) the `STOP` command is now implemented in my current PR #8061 (and the `MOVE` is not a problem since it never gets called); the solution is as follows..

```
PUT /api/shades/{id}
{
  "shade" : {
    "motion" : "stop",
  }
}

```
